content_item_requests_test.go

```diff
@@ -0,0 +1,85 @@
+package main_test
+
+import (
+	"net/http"
+	"net/http/httptest"
+
+	. "github.com/alphagov/publishing-api"
+	. "github.com/alphagov/publishing-api/testhelpers"
```

Comment:
on the lines of code organisation in router, i've added a `integration_test` package that includes testhelpers and integration tests both.